src/Components/TicketSelector/DestinationDesk.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import DeskMenu from "./DeskMenu";
+
+import { GrClose } from "react-icons/gr";
+import { theme } from "../../styles/theme";
+
+class DestinationDesk extends Component {
+  constructor() {
+    super();
+    this.state = {
+      location: {},
+    };
+  }
+
+  componentDidMount() {
+    fetch("/data/data.json")
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          location: res.location,
+        });
+      });
+  }
+
+  closeModal = () => {
+    this.props.closeDestination();
+  };
+
+  render() {
+    const { location } = this.state;
+    const {
+      top,
+      left,
+      closeDestination,
+      selectDepPlace,
+      selectArrPlace,
+    } = this.props;
+
+    return (
+      <Desk top={top} left={left}>
+        <DeskContainer>
+          <Headers>
+            <span>도시 선택</span>
+            <span onClick={this.closeModal}>
+              <GrClose />
+            </span>
+          </Headers>
+          <Main>
+            <MainTop>
+              <input type="text" placeholder="도시명을 입력하세요" />
+              <button>검색</button>
+            </MainTop>
+            <MainBottom>
+              <div className="bottomHeader">주요도시 바로 선택</div>
+              {location.length && (
+                <table>
+                  {location.map((location, idx) => (
+                    <DeskMenu
+                      key={idx}
+                      region={location.region}
+                      locate={location.locate}
+                      closeDestination={closeDestination}
+                      selectDepPlace={selectDepPlace}
+                      selectArrPlace={selectArrPlace}
+                    />
+                  ))}
+                </table>
+              )}
+            </MainBottom>
+          </Main>
+        </DeskContainer>
+      </Desk>
+    );
+  }
+}
+
+const Desk = styled.div`
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.03);
+  top: ${(props) => props.top};
+  left: ${(props) => props.left};
```

Comment:
```suggestion
  top: ${({ top }) => top};
  left: ${({ left }) => left};
```
함수 파라미터 자리에서 직접 구조분해 할당 해줄 수 있습니다!

Reply:
수정했습니다!!